src/repositories/usersRepository.js

```diff
@@ -23,3 +23,11 @@ export const getUserByEmail = async (email) => {
   );
   return user[0];
 };
+
+export const getUsersByName = async (search) => {
+  const { rows: users } = await connection.query(
+    `SELECT id, name, pic_url FROM users WHERE name ILIKE $1||'%'`,
+    [search]
```

Comment:
This query searches for exactly the search param or for anything, maybe searching for users beginning with would be nicer. This can be changed by changing to:

```
`SELECT id, name, pic_url FROM users WHERE name ILIKE $1`,
[`${search}%`]
```